academy/0.0-B9lab-Blockchains/3_managed.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Managed Blockchains"
+order: 3
+description: Introduction to Managed Networks
+tag: fast-track
+---
+
+# Introduction to Managed Networks
+
+Managed networks, just like public networks, rely on blockchain data structures. But unlike public blockchain networks they do not *necessarily* need to mitigate the Byzantine General's Problem because they operate in a predictable environment with elements of authority, hierarchy and accountability.
+
+![Public and Managed Network Comparison](images/00_08_public_vs_private_comparison_dark.png)
+
+# Managed Blockchain Networks
+
+Public networks are based on game theory and economic incentives, which means that every action is probabilistic. We have no guarantee that our transactions will be picked up and even the integrity of the network is merely very likely, not 100% guaranteed.
+
+This is often unacceptable, for example, for traditional financial institutions. Still, one of the biggest expenses financial and other institutions face is the operation and maintenance of infrastructure as well as the cost resulting from leaks, hacks, reconciliation with trading partners, errors and data incompatibility. Blockchain seems like a promising solution. 
+
+Unlike public networks where the interaction between participants is governed by the protocol and crypto-economic incentives, in managed networks the blockchain protocol is a technical enforcement of pre-existing relationships and legally enforceable agreements.
+
+![Public vs Private](images/00_10_public_vs_private_comparison_table.png)
+
+**Private blockchains**:
+
+- **can be designed for a limited number of vetted and approved participants.** Accommodation of performance-challenged and poorly connected nodes is of lesser importance. 
+
+- **can be designed for optimised performance.** Most participants in an Enterprise network will be well-capable of standing up well-connected high-performance and high-availability nodes. A group of participants can agree to raise the bar defining minimum system requirements significantly. For example, participation might be limited to sizable Enterprise-class servers with redundant low-latency, high-bandwidth network connections.
```

Comment:
I'm not sure we need to include a rather long elaboration -starting with "For example, participation...". This makes the bullet point rather long. Maybe you could dive into it in a paragraph following the bullet points.